apps/dashboard/app/models/user_configuration.rb

```diff
@@ -128,4 +133,8 @@ def add_property_methods
       end
     end
   end
+
+    private
+
+    attr_reader :config, :request_hostname
```

Comment:
added `config` as well to make it consistent

Reply:
👍